.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+**Related Issues**
```

Comment:
So this PULL_REQUEST_TEMPLATE is what the person opening the PR should leave as a comment in the PR? Or should this template be submitted as its own file?

Reply:
whenever you open a pr or issue the markdown files replace the default blank field in the request

Reply:
oh cool